### PR TITLE
Accept subscribable on rxLet input

### LINKED ISSUE #1541: `rxLet`: Should accept Subscribable, not only ObservableInput
# Description

I have some non-rxjs code with Subscribable and rxLet directive just do not work because it does not expect it

Basically 

`rxLet: ObservableInput<U> | U | null | undefined;`

### PR DESCRIPTION
**Pull request by AlirezaEbrahimkhani**
resolves #1541

### CODE REVIEW

File: libs/template/let/src/lib/let.directive.ts [hunk 1]
```diff
@@ -130,7 +131,7 @@ export class RxLet<U> implements OnInit, OnDestroy, OnChanges {
    *
    * @param { ObservableInput<U> | U | null | undefined } rxLet
    */
-  @Input() rxLet: ObservableInput<U> | U | null | undefined;
+  @Input() rxLet: ObservableInput<U> | Subscribable<U> | U | null | undefined;
```
[the remark this answers is not in the document]
Comment (AlirezaEbrahimkhani) [reply libs/template/let/src/lib/let.directive.ts:134]: You're right, I will add test cases for that
thanks for your comment

**Comment by eneajaho** [comment]
Hello @AlirezaEbrahimkhani Anything you need to move this PR forward?

**Comment by AlirezaEbrahimkhani** [comment]
@edbzn @eneajaho 
I did the reviews.

**Comment by AlirezaEbrahimkhani** [comment]
Could you please review this PR?
@eneajaho @edbzn

File: libs/template/let/src/lib/tests/let.directive.subscribable.spec.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    expect(fixtureLetDirectiveTestComponent).toBeDefined();
+    expect(letDirectiveTestComponent).toBeDefined();
+    expect(componentNativeElement).toBeDefined();
+  });
+
+  it('should display value from Subscribable', fakeAsync(() => {
+    letDirectiveTestComponent.value$ = 42;
+    fixtureLetDirectiveTestComponent.detectChanges();
+    expect(componentNativeElement.textContent.trim()).toBe('42');
+  }));
```
Comment (hoebbelsB) [inline libs/template/let/src/lib/tests/let.directive.subscribable.spec.ts:71]: this test doesn't test the `subscribable`. You are essentially overwriting the `BehaviorSubject` with a `number` which is getting rendered.

Instead, you should provide an actual `Subscribable`.

```suggestion
  it('should display value from Subscribable', fakeAsync(() => {
    letDirectiveTestComponent.value$ = {
       subscribe: ({ next }) => {
         next(42)
       }
    };
    fixtureLetDirectiveTestComponent.detectChanges();
    expect(componentNativeElement.textContent.trim()).toBe('42');
  }));
```

**Review by hoebbelsB — CHANGES_REQUESTED** [review]
as pointed out in the other comment, please adjust the spec so that it actually tests for a subscribable input

**Comment by hoebbelsB** [comment]
@AlirezaEbrahimkhani any update? :-)

**Comment by AdrianRomanski** [comment]
@hoebbelsB I'll make it from the scratch on my fork

**Comment by hoebbelsB** [comment]
close in favor of https://github.com/rx-angular/rx-angular/pull/1721

thank u @AlirezaEbrahimkhani for the initial contribution!

**Pull request closed without merging**